react/features/overlay/components/OverlayContainer.js

```diff
@@ -95,6 +121,25 @@ class OverlayContainer extends Component {
     }
 
     /**
+     * Initializes a new ReloadTimer instance.
+     *
+     * @param {Object} props - The read-only properties with which the new
+     * instance is to be initialized.
+     * @public
+     */
+    constructor(props) {
+        super(props);
+        this.state = {
+            /**
+             * Indicates whether the film strip only mode is enabled or not.
+             *
+             * @type {boolean}
+             */
+            filmStripOnly: interfaceConfig.filmStripOnly
```

Comment:
Then wouldn't it be a prop? At any rate, this line will need to change anyway, so I wouldn't add it to the state now.